onnx/defs/parser.cc

```diff
@@ -106,6 +106,32 @@ Status OnnxParser::Parse(char open, IdList& idlist, char close) {
   return Status::OK();
 }
 
+Status OnnxParser::Parse(IdList& idlist, AttrList& attrlist) {
+  idlist.Clear();
+  attrlist.Clear();
+  do {
+    std::string id;
+    ParseIdentifier(id);
+    auto next = NextChar();
+    if (next == ':' || next == '=')
+      Parse(*attrlist.Add(), id);
```

Comment:
Should we forbid the use of attribute without default values after one attribute with a default value?

Reply:
Does order matter for attributes?

Reply:
No, I think it is better to leave the order be immaterial. So, okay to be more lenient than strict here, IMO.

Reply:
> Does order matter for attributes?

not in the proto. So w.r.t. @xadupre 's comment, I agree with @gramalingam  that we do not (or cannot) check for this rule.